src/views/Stack/StackItem.tsx

```diff
@@ -109,6 +109,15 @@ export default class StackItem extends React.PureComponent<Props> {
       renderScene,
     } = this.props;
 
+    const {
+      descriptor: {
+        options: {
+          headerStyleInterpolator: customHeaderStyleInterpolator,
+          cardStyleInterpolator: customCardStyleInterpolator,
+        },
+      },
+    } = scene;
```

Comment:
`options` cannot be `undefined`? I'd rather check it.

Reply:
To be honest I can't determine it by looking at the types, but since all its props are optional, this might be the case here 😅.

Reply:
If TS doesn't complain then it should be okay.